Dockerfile

```diff
@@ -31,16 +31,27 @@ RUN mkdir -p /app/entities
 
 ################# DEVELOPMENT ####################################
 FROM base as development
-COPY . .
+COPY wsgi.py requirements.txt requirements_dev.txt plugins.yml ./
+COPY app app
+COPY entities entities
+COPY python_mapping_plugins python_mapping_plugins
 
 # Run static security check and linters
 RUN pip install -q --trusted-host pypi.org --trusted-host files.pythonhosted.org -r requirements_dev.txt
-RUN pre-commit run --all-files  \
-  && safety check -r requirements.txt -r requirements_dev.txt
+#RUN pre-commit run --all-files  \
+#  && safety check -r requirements.txt -r requirements_dev.txt
 
 # Run pytest with code coverage
 # RUN pytest --cov app
 
+# For debugging: install oteapi locally
+#COPY oteapi-core oteapi-core
+#RUN pip install -U /app/oteapi-core
```

Comment:
Installing for development of oteapi-core needs pip install -e . on volume mounted.